.env.test

```
@@ -72,6 +72,10 @@ SIGNOUT_RATE_LIMIT_MAX_REQUESTS=10
 SIGNOUT_RATE_LIMIT_INTERVAL="1 minute"
 SIGNOUT_ALL_RATE_LIMIT_MAX_REQUESTS=5
 SIGNOUT_ALL_RATE_LIMIT_INTERVAL="1 minute"
+OAUTH_SOCIAL_INITIATE_RATE_LIMIT_MAX_REQUESTS=10
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OAUTH_SOCIAL_INITIATE_RATE_LIMIT_MAX_REQUESTS key should go before the OAUTH_TOKEN_RATE_LIMIT_INTERVAL key

(UnorderedKey)